File: submissions/sm_036_prabhat/week_03/day_3/session_2/count_vowels.js
function countVowel (vowel) {
    var count = 0;
    for(var i = 0;i < vowel.length; i++) {
        if(vowel[i] == "a" || vowel[i] == "e" || vowel[i] == "i" || vowel[i] == "o" || vowel[i] == "u") {
            count ++;
        }
    }
    return count;
}

console.log(countVowel("prabhatranjan"));

File: submissions/sm_102_amit/week_14/day_3/evaluation/count_vowels.py
# Count vowel
string = input("Enter your string: ")
lower_vowel = ["a", "e", "i", "o", "u"]
upper_vowel = ["A", "E", "I", "O", "U"]
l_count = 0
u_count = 0
for x in string:
    if x in lower_vowel:
        l_count += 1
    elif x in upper_vowel:
        u_count += 1
print("Input: ", string)
print("Lower Vowel: ", l_count)
print("Upper Vowel: ", u_count)

# Sample Case:
'''
Enter your string: Amit Kumar
Input:  Amit Kumar
Lower Vowel:  3
Upper Vowel:  1
'''
